sdk/ml/azure-ai-ml/azure/ai/ml/operations/_data_operations.py

```diff
@@ -314,6 +318,54 @@ def create_or_update(self, data: Data) -> Data:
                     )
             raise ex
 
+    def import_data(self, data_import: DataImport) -> Job:
+        """Returns the data import job that is creating the data asset.
+
+        :param data_import: DataImport object.
+        :type data_import: azure.ai.ml.entities.DataImport
+        :return: data import job object.
+        :rtype: ~azure.ai.ml.entities.Job
+        """
+
+        connection_name = data_import.source.connection.split(':')[-1]
+        experiment_name = "data_import_" + data_import.name
+        display_name = experiment_name + "_" + connection_name
+        import_job = import_data(
+            description=display_name,
+            display_name=display_name,
+            experiment_name=experiment_name,
+            compute="serverless",
+            source=data_import.source,
+            outputs={"sink": Output(type=data_import.type, path=data_import.path, name=data_import.name)},
+            is_deterministic=False,
+            component="azureml://registries/azureml-dev/components/import_data_database/versions/1"
```

Comment:
Does database and filesystem use the same component？

Reply:
Good catch. Fixed